openlibrary/plugins/openlibrary/js/SearchBar.js

```diff
@@ -13,6 +13,13 @@ const FACET_TO_ENDPOINT = {
     text: '/search/inside',
 };
 const DEFAULT_FACET = 'all';
+const DEFAULT_JSON_FIELDS = [
```

Comment:
Does this mean we wouldn't have `ia` list or `languages`? Those two may be useful in the future